Packs/Code42/Integrations/Code42/README.md

```diff
@@ -277,22 +264,60 @@ There are no input arguments for this command.
 | Code42.DepartingEmployee.UserID | string | Internal Code42 User ID for the Departing Employee. | 
 | Code42.DepartingEmployee.Username | string | The username of the Departing Employee. | 
 | Code42.DepartingEmployee.Note | string | Note associated with the Departing Employee. | 
-| Code42.DepartingEmployee.DepartureDate | unknown | The departure date for the Departing Employee. | 
+| Code42.DepartingEmployee.DepartureDate | Unknown | The departure date for the Departing Employee. | 
 
 
 #### Command Example
 ```!code42-departingemployee-get-all```
 
-#### Human Readable Output
+#### Context Example
+```
+{
+    "Code42": {
+        "DepartingEmployee": [
+            {
+                "DepartureDate": null,
+                "Note": "test",
+                "UserID": "921333907298179098",
+                "Username": "user1@example.com"
+            },
+            {
+                "DepartureDate": "2020-07-20",
+                "Note": "This is added using csv file to test bulk adding of users to high risk employee list",
+                "UserID": "948333588694228306",
+                "Username": "user2@example.com"
+            },
+            {
+                "DepartureDate": null,
+                "Note": "",
+                "UserID": "912211111144144039",
+                "Username": "user3@example.com"
+            }
+        ]
+    }
+}
+```
 
-| **UserID** | **DepartureDate** | **Note** | **Username** |
-| --- | --- | --- | --- |
-| 123 | 2020-02-28 | Leaving for competitor | john.user@example.com |
+#### Human Readable Output
+### All Departing Employees
+|DepartureDate|Note|UserID|Username|
+|---|---|---|---|
+|  | test | 921286907298179098 | user1@example.com |
+| 2020-07-20 | This is added using csv file to test bulk adding of users to high risk employee list | 948938588694228306 | sagar.patel+l2@code42.com |
```

Comment:
There are some real people's names/email addresses in here, should probably anonamize